src/go/NOTES.md

```diff
@@ -2,6 +2,6 @@
 
 ## OS Support
 
-This Feature should work on recent versions of Debian/Ubuntu-based distributions with the `apt` package manager installed.
+This Feature should work on recent versions of Debian/Ubuntu, RedHat Enterprise Linux, Fedora, Alma, and RockyLinux distributions with the apt, yum, dnf, or microdnf package manager installed.
```

Comment:
Can we also bump the `minor` version for this Feature? Thanks!

Reply:
Commit c9c6aaa bumps version to 1.3.0